Libplanet.Extensions.Cocona/Commands/ApvCommand.cs

```diff
@@ -263,7 +267,47 @@ void TreeIntoTable(IValue tree, List<(string, string)> table, string key)
                 TreeIntoTable(v.Extra, data, "extra");
             }
 
-            Utils.PrintTable(("Field", "Value"), data);
+            if (json)
+            {
+                Console.WriteLine(JsonSerializer.Serialize(data.ToDictionary(e => e.Item1, e => e.Item2)));
+            }
+            else
+            {
+                Utils.PrintTable(("Field", "Value"), data);
+            }
+        }
+
+        [Command(Description = "Query app protocol version (a.k.a. APV) of target node.")]
+        public void Query(
+            [Argument(
+                Name = "TARGET",
+#pragma warning disable MEN002 // Line is too long
+                Description = "Comma seperated peer information({pubkey},{host},{port}>) of target node. (e.g. 027bd36895d68681290e570692ad3736750ceaab37be402442ffb203967f98f7b6,9c.planetarium.dev,31236")]
```

Comment:
```suggestion
                Description = "Comma seperated peer information({pubkey},{host},{port}>) of target node. (e.g. 027bd36895d68681290e570692ad3736750ceaab37be402442ffb203967f98f7b6,9c.planetarium.dev,31236)")]
```

And I wonder what means `>` in `{pubkey},{host},{port}>`.

Reply:
It's just a typo. 😅